metaci/build/models.py

```diff
@@ -410,9 +418,23 @@ def run(self):
             )
             if org_config.created:
                 self.delete_org(org_config)
+
             self.logger = init_logger(self)
             self.logger.error(str(e))
             self.delete_build_dir()
+            if (
+                self.plan.change_traffic_control
+            ):
+                try:
+                    send_stop_webhook(
+                        self.release,
+                        self.plan.role,
+                        self.org.configuration_item,
+                        "Failed - no impact",
+                    )
+                except Exception as err:
+                    message = f"Error while sending implementation stop step webhook: {err}"
+                    self.logger.error(message)
```

Comment:
Let's log the message from the error that was raised in `send_stop_webhook`:

```suggestion
                    self.logger.error(str(err))
```